<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="AxisView" name="yAxisView" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="AxisView" name="xAxisView" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="Plotter" name="plotter" native="true"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="LegendView" name="legend">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLines"/>
   <addaction name="actionDots"/>
   <addaction name="actionMarkers"/>
  </widget>
  <action name="actionLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Линии</string>
   </property>
  </action>
  <action name="actionDots">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Точки</string>
   </property>
  </action>
  <action name="actionMarkers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Маркеры</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AxisView</class>
   <extends>QWidget</extends>
   <header>axisview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LegendView</class>
   <extends>QListView</extends>
   <header>legendview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
